analyzers/tests/SonarAnalyzer.UnitTest/Rules/UriShouldNotBeHardcodedTest.cs

```diff
@@ -64,7 +64,7 @@ public class UriShouldNotBeHardcodedTest
 
         [DataTestMethod]
         [DataRow("2.0.4", "2.0.3", "2.1.1")]
-        [DataRow(Constants.NuGetLatestVersion, Constants.NuGetLatestVersion, Constants.NuGetLatestVersion)]
```

Comment:
will you open a ticket to bring this back to NuGet Latest?

why: we keep LatestVersion to be aware if some API changes lead to our rules creating FPs or FNs  (e.g. a deprecated attribute might be removed, so we won't raise anymore on net8 and we need to update the rule)

Reply:
I created the ticket #8363

The goal would be to revert this PR when changing the target framework of the UT project to `net8.0`